### DISCUSSION TITLE
[Feature Request] Firebase Firestore

### DISCUSSION THREAD

**Discussion by dpnnl** [invertase/flutterfire_desktop#77] [Ideas]
Hi!
I would like to know if they could implement Firebase Firestore, it would be extremely useful for apps on Windows and Linux, it would be interesting too if an alpha version were made available, if it is already in development.

**Comment by pr-Mais** [comment]
It's possible for sure, but we're starting with smaller ones first, see the [Roadmap](https://github.com/invertase/flutterfire_desktop/issues/52), in [this comment](https://github.com/invertase/flutterfire_desktop/issues/52#issuecomment-1070870967) I mentioned why. There's currently no estimation on when the work on Firestore will start, but hopefully we get to it soon.

**Reply by melWiss** [reply]
Is it okay to start working on Firestore native dart implementation by myself? I believe it would accelerate a little since Firestore is one of the most demanded services of Firebase.
